src/openassetio-core/CMakeLists.txt

```diff
@@ -41,13 +41,16 @@ set_target_properties(openassetio-core PROPERTIES OUTPUT_NAME openassetio)
 # Add to the set of installable targets.
 install(TARGETS openassetio-core EXPORT ${PROJECT_NAME}_EXPORTED_TARGETS)
 
-if(WIN32)
```

Comment:
[consider] Splitting these build changes into a separate PR (or the previous one?) as this one seems CI related otherwise.

Reply:
Haha sorry yes.

Reply:
Moved the CMakeLists changes into the previous (i.e. first) commit e4fdb46e